### PR TITLE
Upgrade cypress, fix and stabilize tests

### PR DESCRIPTION
**Pull request by macobo**
## Changes

- Upgrade cypress
- Turn on retries by default 
- Fix broken command palette spec

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

File: cypress/integration/commandPalette.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
     it('Shows on toggle button click', () => {
         cy.get('[data-attr=command-palette-toggle]').click()
         cy.get('[data-attr=command-palette-input]').should('exist')
     })
 
     it('Shows on Ctrl + K press', () => {
-        cy.get('body').type('{ctrl}C')
+        cy.get('body').type('{ctrl}k')
         cy.get('[data-attr=command-palette-input]').should('exist')
-        cy.get('body').type('{cmd}C')
```
Comment (Twixes) [inline cypress/integration/commandPalette.js:14]: Wow, this was dumb.

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -115,6 +115,8 @@ jobs:
                   # Recommended: pass the GitHub token lets this action correctly
                   # determine the unique run id necessary to re-run the checks
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  # Tolerate flaky tests
+                  CYPRESS_RETRIES: 3
```
Comment (Twixes) [inline .github/workflows/e2e.yml:119]: Shame that this is needed.

**Review by Twixes — APPROVED** [review]

File: .github/workflows/e2e.yml (hunk 2 above)
Comment (macobo) [reply .github/workflows/e2e.yml:119]: In my experience, this is somewhat typical in tests that deal with UI, but agreed.

File: cypress/integration/commandPalette.js (hunk 1 above)
Comment (macobo) [reply cypress/integration/commandPalette.js:14]: :sweat_smile: took me like 5 minutes to figure out what the "bug" is.

**Pull request merged**